### ISSUE TITLE
transmission should use all of the machine's IP addresses

### ISSUE THREAD

**Issue by SoniEx2** [transmission/transmission#916]
currently it seems transmission picks one address on IPv4 and IPv6 and doesn't bother with the rest. this can cause undebuggable connection issues. please use all IPs instead.

**Comment by SoniEx2** [comment]
additionally the "test port" feature should list all IPs and whether the port is open for each of the IPs.

**Comment by snixel** [comment]
I agree, I also noticed on IPv6 only the TCP is open. UDP stays closed on IPv6 even when uTP is enabled, while on ipv4 both ports are open.

**Comment by TigerP** [comment]
it seems to me that transmission should not always use all IP addresses, when the bind-address-ipv[46] are set to a specific address.

**Comment by SoniEx2** [comment]
honestly, those should accept multiple IPs too.
